ballerina/tests/58_nested_message_nested_enum_pb.bal

```diff
@@ -0,0 +1,224 @@
+
```

Comment:
Shall we add license headers

Reply:
Added in https://github.com/ballerina-platform/module-ballerina-grpc/pull/378/commits/91bf77131917ccabe97fdd7123d2c60ee350f348